docs/user_guide/queue_config.md

```diff
@@ -191,7 +192,18 @@ partitions:
 The placement rules are defined and documented in the [placement rule](placement_rules.md) document.
 
 Each partition can have only one set of placement rules defined. 
-If no rules are defined the placement manager is not started and each application *must* have a queue set on submit.  
+If no rules are defined the placement manager is not started and each application *must* have a queue set on submit.
+
+### Statedump filepath
+
+The statedump filepath defines the output file for YuniKorn statedumps. It is optionally set on the partition level. If set,
```

Comment:
NIT: defines the output files for YuniKorn state dumps

there can be more than one file, correct? Do we need to briefly talk about the rotation policy here as well?

Reply:
Thanks for pointing that out -- added details about the rotation policy below.

There is only one file that is being written to for statedumps as per the log rotation policy. Different partitions can technically have different values, but statedumps contain information across multiple partitions. As a result, only the first occurrence of `statedumpfilepath` takes precedence for defining the location of the output file, even if other partitions are configured with different values. If no occurrence is found, it will default to `yunikorn-state.txt` (previous behavior).

Reply:
Thanks for the clarification, it helps!